packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testOpenPreferences.st

```diff
@@ -0,0 +1,16 @@
+as yet unclassified
+testOpenPreferences
+
+	| prefBrowser prefBrowserMorph |
+
+	klass new openPreferences.
+
+	prefBrowserMorph := [World submorphs detect: [ :morph | morph class name = #PreferenceBrowserMorph]]
+		on: NotFound
+		do: [:exc | Transcript show: exc messageText; cr. ^nil].
```

Comment:
you could write `self assert: false description 'description'` and provide a proper description of the problem since there is no real assert to understand the problem.